firstvoices/backend/resources/utils/import_export_widgets.py

```diff
@@ -1,6 +1,8 @@
 from django.contrib.auth import get_user_model
 from import_export.widgets import ForeignKeyWidget, Widget
 
+DUMMY_USER_EMAIL = "test@test.com"
```

Comment:
This might eventually be visible to team members-- should we maybe use something like support@fpcc.ca as the dummy account?